### ISSUE TITLE
Torrentio - Automatically delete torrents from RD list

### ISSUE THREAD

**Issue by Jackalon** [TheBeastLT/torrentio-scraper#103]
Whenever a link is selected, it automatically adds them to the RD Torrent List.

Is there any way to automatically delete torrents from RD list, similar to what Kodi addons already do?

**Comment by TheBeastLT** [comment]
No, I don't think that this can be added to torrentio, due to the different way addon system works in stremio. All addons in stremio are remote, deployed on servers, and whenever you need to resolve a url to a RD download link it has to go through the server via redirecting. Also whenever you seek in already watched video it also has to go through the server, since the players use the original url, not the final redirect url.
Another point is efficiency, since this is a remote server that many user make calls to, it has to be efficient and make least amount of api calls to RD servers, so it makes sense to keep the created torrent and just resolve the url instead of creating the torrent each time (ie watching multi season torrent, or continuing a movie the next day etc). Some  people also just add links through torrentio and then watch them elsewhere through RD or some other app. 
Another point is that addons don't know when you've finished watching a video, so it cannot clean up after you finished watching smth, as the addons are not informed about it.
Also there's no limit for torrent links in RD, so  personally I don't see the benefit of this, where as leaving them has clear benefits outlined above.
